src/backends/kimchi/mod.rs

```diff
@@ -302,6 +307,7 @@ impl Backend for KimchiVesta {
         var
     }
 
+    #[allow(unused_variables)]
```

Comment:
remove

Reply:
btw, I have another smaller PR #109 that fixes unused vars separately, if you'd like to see if those changes are reasonable

Reply:
removed in [96f903a](https://github.com/zksecurity/noname/pull/100/commits/96f903af9776c17cc785dadafc2d33382d02856d)